presto-server-rpm/src/main/rpm/postremove

```diff
@@ -3,9 +3,10 @@
 # if this is the last version of presto-server-rpm being removed (i.e. not on upgrade)
 if [ "$1" -eq 0 ]
 then
-    # Delete the conf directory manually during uninstall.
-    # rpm -e wont remove it, because this directory was manually updated in postinstall
-    rm -rf /etc/presto
+    # Delete /etc/presto/env.sh manually during uninstall.
+    # rpm -e will not remove since it's not considered a configuration file
+    # and contains values generated during installation
+    rm -f /etc/presto/env.sh
```

Comment:
What about removing `/etc/presto` if it is empty? (I know it is not empty today because of remaining `node.properties`.

Reply:
The empty directory check is easy enough and related to this PR. I'll add that, then hopefully consider these changes for the PR complete. 

The RPM check inside a Docker container is a good idea and it should be pursued. There are some tools like testinfra/serverspec that can be used to validate that a node is setup with expected values. I'll setup an issue to investigate/implement those tests.